python/ray/tests/test_runtime_env_complicated.py

```diff
@@ -180,6 +181,65 @@ def test_get_conda_env_dir(tmp_path):
         assert (env_dir == str(tmp_path / "envs" / "tf2"))
 
 
+def test_conda_create_task(shutdown_only):
+    """Tests dynamic creation of a conda env in a task's runtime env."""
```

Comment:
Can we add a test for the case when the conda env fails to install? Ideally we should eventually get a failure and there should definitely be some meaningful errors streamed to the driver.